config/locales/translations/zendesk_apps_support.yml

```diff
@@ -412,3 +412,7 @@ parts:
       key: "txt.apps.admin.error.app_build.oauth_parameter_cannot_be_secure"
       title: "oauth parameter cannot be set to be secure."
       value: "oauth parameter cannot be set to be secure."
+  - translation:
+      key: "txt.apps.admin.error.app_build.invalid_url"
+      title: "App builder job: this value needs to be a valid URL, but something else was passed in. placeholder value is taken from user input. You can translate as: The value %{field} must be a valid URL... to avoid any gender issues."
```

Comment:
This title doesn't make much sense to me 😅, generally they should just be a short description of what the error means. In this case  something like 'The url provided was not valid' should do.

Reply:
I just copied the invalid_Sstring translation 😅. Happy to change it

Reply:
I see. Happy to leave it as is since there's a few with a similar description already. Just seems pretty weird to me that we mention the app builder job here - that's in a completely different code base 🤷

Reply:
maybe some historical issue